Backoffice/Security/Authorization/Voter/ContentVoter.php

```diff
@@ -36,12 +36,13 @@ protected function supportSubject($subject)
      * A user can read a content if it is in his perimeter
      *
      * @param ContentInterface|string $subject
-     * @param UserInterface           $user
+     * @param TokenInterface          $token
      *
      * @return bool
      */
-    protected function voteForReadAction($subject, UserInterface $user)
+    protected function voteForReadAction($subject, TokenInterface $token)
     {
+        $user = $token->getUser();
```

Comment:
@alavieille : there's no more the need to create the local var $user, you can directly inject $token->getUser() in the condition, or better, you can inject the $token in `isSubjectInPerimeter`. That way if we refacto the token to store the perimeter, the modification here will be small